Makefile

```diff
@@ -9,8 +9,8 @@ install:
 
 .PHONY: format
 format:
-	$(autoflake)
-	$(isort)
+#	$(autoflake)
+#	$(isort)
```

Comment:
Can you make an issue for this? We might just need to pick the right options or file an issue upstream.